### PR TITLE
Split backup script to make one part overridable

### PR DESCRIPTION
**Pull request by raneq**
We use restic to manage the "snapshots" but we want to give restic what we want, not single folders.
We want to have a default template, but not to force it.

### CODE REVIEW

**Review by enricostano — APPROVED** [review]
:clap:

**Comment by raneq** [comment]
When this is merged we can add a tag for version 0.1.0 or 1.0.0 (do we use some criterion like "[semantic versioning](https://semver.org/#faq)"?

**Comment by enricostano** [comment]
> When this is merged we can add a tag for version 0.1.0 or 1.0.0 (do we use some criterion like "semantic versioning"?

Yes, we should start using https://github.com/coopdevs/backups-role/releases for this repo too and publish it to Ansible Galaxy.

File: templates/cron-prepare.sh.j2 [hunk 1]
```diff
@@ -14,11 +14,23 @@ set -e
...
+## Postgres DB dump ##
+######################"
 /usr/bin/pg_dump --encoding=UTF8 {{ backups_role_db_name }} > {{ backups_role_tmp_path }}/pg_dump
+echo "...done"
 
-echo "## Compress configuration files ##"
+echo "
+##################################
+## Compress configuration files ##
+##################################"
```
Comment (enricostano) [inline templates/cron-prepare.sh.j2]: Too long :wink:

File: templates/cron-upload.sh.j2 [hunk 2]
```diff
@@ -3,17 +3,36 @@
...
+## Restic check ##
+##################"
 restic check
+echo "...done"
 
-echo "## Remove temporal files ##"
+echo "
+###########################
+## Remove temporal files ##
+###########################"
```
Comment (enricostano) [inline templates/cron-upload.sh.j2]: Too long :wink:

**Review by danypr92 — APPROVED** [review]
:clap:

**Pull request merged**